### PR TITLE
Remove ROS2 native connection

### PR DESCRIPTION
**Pull request by achim-k**
**User-Facing Changes**
- Remove ROS2 native connector

**Description**
Removes the ROS2 native connector in favor of the `foxglove_bridge` / `rosbridge` connections.

See also #4900

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2490,7 +2477,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@foxglove/rtps@npm:1.6.0, @foxglove/rtps@npm:^1.6.0":
+"@foxglove/rtps@npm:1.6.0":
```
Comment (jtbandes) [inline yarn.lock:2480]: I think we should be able to completely remove @foxglove/rtps as well?

**Review by jtbandes — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (defunctzombie) [reply yarn.lock:2480]: 🔪

Comment (jtbandes) [reply yarn.lock:2480]: Theoretically lint-dependencies will catch this.

Comment (jtbandes) [reply yarn.lock:2480]: image

**Pull request merged**